src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -109,7 +122,7 @@ trait Interpreter {
 
     case t: Transformer[_, _] if t.transformationReady => t.function()
 
-    case _ => null // this means the node cannot be evaluated
+    case v: SValue  => specificTransformations(context, v)
```

Comment:
My current understanding is that specificTransformations should be done before evaluateNode is called.
So here we will just waste CPU cycles.
It is better to call `super.evaluateNode(...)`, because rewriting in Scala usually implemented using "stackable traits" pattern

Reply:
@ergomorphic done, specificTransformations eliminated